src/Service/PageTitle.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Service;
+
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class PageTitle
+{
+    public function __construct(
+        private readonly BreadcrumbTrail $breadcrumbTrail,
+        private readonly Fallbacks $fallbacks,
+        private readonly TranslatorInterface $translator,
+    ) {
+    }
+
+    public function getTitle(): string
+    {
+        $breadcrumbs = $this->breadcrumbTrail->all();
+        $breadcrumbs =array_reverse($breadcrumbs);
+
+        if (count($breadcrumbs) === 0) {
+            return $this->fallbacks->get('site_title');
+        }
+
+        $title = '';
+        foreach ($breadcrumbs as $breadcrumb) {
+            $title .= $this->translator->trans($breadcrumb->getTitle()).' - ';
```

Comment:
code styling, missing spaces after and before `.`

Reply:
Isn't `implode(array_reverse($this->breadcrumb->all())) . $this->fallbacks->get('site_title');` more logical?